src/main/java/it/gov/pagopa/payhub/activities/mapper/ingestionflow/organization/OrganizationMapper.java

```diff
@@ -0,0 +1,38 @@
+package it.gov.pagopa.payhub.activities.mapper.ingestionflow.organization;
+
+import it.gov.pagopa.payhub.activities.dto.ingestion.organization.OrganizationIngestionFlowFileDTO;
+import it.gov.pagopa.pu.organization.dto.generated.OrganizationRequestBody;
+import it.gov.pagopa.pu.organization.dto.generated.OrganizationStatus;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OrganizationMapper {
+
+  public OrganizationRequestBody map(OrganizationIngestionFlowFileDTO dto,Long brokerId) {
+
+    return OrganizationRequestBody.builder()
+        .ipaCode(dto.getIpaCode())
+        .orgFiscalCode(dto.getOrgFiscalCode())
+        .orgName(dto.getOrgName())
+        .orgTypeCode(dto.getOrgTypeCode())
+        .orgEmail(dto.getOrgEmail())
+        .postalIban(dto.getPostalIban())
+        .iban(dto.getIban())
+        .password(dto.getPassword().getBytes())
+        .segregationCode(dto.getSegregationCode())
+        .cbillInterBankCode(dto.getCbillInterBankCode())
+        .orgLogo(dto.getOrgLogo())
+        .status(OrganizationStatus.valueOf(dto.getStatus()))
+        .additionalLanguage(dto.getAdditionalLanguage())
+        .startDate(dto.getStartDate().toLocalDate())
+        .brokerId(brokerId)
+        .ioApiKey(dto.getIoApiKey().getBytes())
+        .sendApiKey(dto.getSendApiKey().getBytes())
```

Comment:
these are wrong! you cannot do that...
ioApiKey and sendApiKey are encrypted! you cannot fill them using getBytes.. you should encrypt the input data...
but activities should not know about the encryption key...

we should set these fields using the UPDATE API, or we should expose a new POST operation to accept the entire DTO with plain values and cipher them after...